wix-embedded-mysql/src/main/java/com/wix/mysql/ClassPathScriptResolver.java

```diff
@@ -0,0 +1,51 @@
+package com.wix.mysql;
+
+import java.io.File;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.List;
+
+import static java.lang.String.format;
+
+/**
+ * Helper for locating schema init scripts in a classpath
+ *
+ * @author viliusl
+ * @since 06/06/15
+ */
+public class ClassPathScriptResolver {
+
+    /**
+     * Locates a single file in a classpath, ex. 'db/init_schema.sql'
+     */
+    public static File file(final String path) {
+        String normalizedPath = path.startsWith("/") ? path : format("/%s", path);
+        URL resource = ClassPathScriptResolver.class.getResource(normalizedPath);
```

Comment:
not sure what we are doing here, but guava has some api's that does exactly the same

```java
            URL resource = Resources.getResource(ClassPathScriptResolver.class, normalizedPath);
            return new File(resource.getFile());
```